### DISCUSSION TITLE
Make default color palettes go to darker tones

### DISCUSSION THREAD

**Discussion by nicoszerman** [tailwindlabs/tailwindcss#8054] [Ideas]
I am developing the website for my startup and I was trying to set a dark background color with contrasting light (similar to [stripe.com](https://stripe.com/) if you scroll to "Designed for developers"). However, after trial and error it was sad for me to find out that the darkest tones of the included color palettes are actually pretty light. This is hard to notice in the official website (https://tailwindcss.com/docs/customizing-colors) because the lighter colors to the left serve as contrast points, facilitating an optical illusion of dark tone, but if you look at the darkest tones of red, green, blue, etc.. you will notice they are not so dark. Except for the gray-scales (slate, zinc, neutral, stone), the 900s are too light to be able to use contrasting light text against them.

How about adding sky-1000, rose-1000, or a set of dark tones maybe dark-sky, dark-rose, etc.. ?

In particular, adding sky-1000, sky-1100, etc.. looks like a very non-intrusive addition that facilitates what seems to me an expected feature: dark color tones.

**Comment by matthewpull** [comment]
I completely agree with this. On the light end the 50 series is a nice stepping point between white and 100, whether that be as a background, hover state, etc.

There isn't a dark mode equivalent - the difference between something like gray-900 (or, in my case, zinc-900) and black is staggering.

Personally, I'd be more in favour of calling these 950 rather than 1000 - I'd always considered 0 as pure white and 1000 as pure black, so 950 would make more sense imo - but the nomenclature isn't a major issue for me.

**Comment by nicoszerman** [comment]
The distance between 900 and black is much greater than between white and
100. Adding 950 colors would break the “linear” scale represented by white,
50, 100, 200, …, 900. It seems to me dark colors should have their own
color labels or start numbering past 1000.

On Tue, May 17, 2022 at 6:27 PM Matthew Pull ***@***.***>
wrote:

> I completely agree with this. On the light end the 50 series is a nice
> stepping point between white and 100, whether that be as a background,
> hover state, etc.
>
> There isn't a dark mode equivalent - the difference between something like
> gray-900 (or, in my case, zinc-900) and black is staggering.
>
> Personally, I'd be more in favour of calling these 950 rather than 1000 -
> I'd always considered 0 as pure white and 1000 as pure black, so 950 would
> make more sense imo - but the nomenclature isn't a major issue for me.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tailwindlabs/tailwindcss/discussions/8054#discussioncomment-2771242>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADLB3ZE5VE3S5UGLRIGDPJ3VKQMLVANCNFSM5WGMKTMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 

Nic Szerman

***@***.***

College of Engineering, Class of 2021

University of Michigan, Ann Arbor, Michigan

**Reply by matthewpull** [reply]
Gotta say that I'm not much of a fan of having separate labels - I don't see why a red that's a bit darker than the current red-900 would be called dark-red, that seems the break all the conventions we currently have.

But it seems like we're getting caught up over the name - that's such a minor point and definitely up to the Tailwind team to decide what they think best suits their system.

Personally, all I want is the darker colors, that's the goal here, regardless of what terminology we end up going for.

**Comment by nicoszerman** [comment]
Agreed!

On Tue, May 17, 2022 at 6:45 PM Matthew Pull ***@***.***>
wrote:

> Gotta say that I'm not much of a fan of having separate labels - I don't
> see why a red that's a bit darker than the current red-900 would be called
> dark-red, that seems the break all the conventions we currently have.
>
> But it seems like we're getting caught up over the name - that's such a
> minor point and definitely up to the Tailwind team to decide what they
> think best suits their system.
>
> Personally, all I want is the darker colors, that's the goal here,
> regardless of what terminology we end up going for.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tailwindlabs/tailwindcss/discussions/8054#discussioncomment-2771320>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADLB3ZBEBWO3MRSNFSLDMR3VKQOPFANCNFSM5WGMKTMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 

Nic Szerman

***@***.***

College of Engineering, Class of 2021

University of Michigan, Ann Arbor, Michigan

**Comment by nicoszerman** [comment]
Happy to see that after one year you guys did add the 950 color option. Thank you!
